apps/gitness/src/pages/profile-settings/profile-settings-general-container.tsx

```diff
@@ -26,7 +28,10 @@ export const SettingsProfileGeneralPage: React.FC = () => {
         setUserData({
           name: data.display_name || '',
           username: data.uid || '',
-          email: data.email || ''
+          email: data.email || '',
+          language: localStorage.getItem('i18nextLngSystem')
```

Comment:
Try not to repeat localStorage.getItem('i18nextLngSystem') and localStorage.getItem('i18nextLng') multiple times?